pantheon-bundle/frontend/src/app/__snapshots__/product.test.tsx.snap

```diff
@@ -2,97 +2,97 @@
 
 exports[`Product tests should render Product component 1`] = `
 <Fragment>
-  <Bullseye>
-    <Form>
-      <div
-        className="app-container"
-      >
-        <div>
-          <FormGroup
-            fieldId="product_name"
+  <Form
+    className="p2-product__form"
+  >
+    <div
+      className="app-container"
+    >
+      <div>
+        <FormGroup
+          fieldId="product_name"
+          isRequired={true}
+          label="Product Name"
+        >
+          <ForwardRef
+            id="product_name_text"
             isRequired={true}
-            label="Product Name"
-          >
-            <ForwardRef
-              id="product_name_text"
-              isRequired={true}
-              onChange={[Function]}
-              placeholder="Product Name"
-              type="text"
-              value=""
-            />
-          </FormGroup>
-          <br />
-          <FormGroup
-            fieldId="product_url_fragment"
-            isRequired={true}
-            label="Product URL Fragment"
-          >
-            <ForwardRef
-              id="product_url_fragment_text"
-              isRequired={true}
-              onChange={[Function]}
-              placeholder="URL Fragment"
-              type="text"
-              value=""
-            />
-          </FormGroup>
-          <br />
-          <FormGroup
-            fieldId="product_description"
-            label="Product Description"
-          >
-            <ForwardRef
-              id="product_description_text"
-              onChange={[Function]}
-              placeholder="Product Description"
-              type="text"
-              value=""
-            />
-          </FormGroup>
-          <br />
-          <FormGroup
-            fieldId="version_name"
-            isRequired={true}
-            label="Product Version:"
-          >
-            <ForwardRef
-              id="new_version_name_text"
-              onChange={[Function]}
-              placeholder="Product Version"
-              type="text"
-              value=""
-            />
-          </FormGroup>
-          <br />
-          <FormGroup
-            fieldId="version_url_fragment"
+            onChange={[Function]}
+            placeholder="Product Name"
+            type="text"
+            value=""
+          />
+        </FormGroup>
+        <br />
+        <FormGroup
+          fieldId="product_url_fragment"
+          isRequired={true}
+          label="Product URL Fragment"
+        >
+          <ForwardRef
+            id="product_url_fragment_text"
             isRequired={true}
-            label="Version URL Fragment:"
+            onChange={[Function]}
+            placeholder="URL Fragment"
+            type="text"
+            value=""
+          />
+        </FormGroup>
+        <br />
```

Comment:
I dislike that we're using `<br>`s here for layout (as opposed to text formatting), but I know it was there before. If you feel bored and want to clean it up, a CSS like making the elements `display: block` or something would be preferred.

Reply:
Since it's far from the point of this PR, we could make a low priority bug ticket?